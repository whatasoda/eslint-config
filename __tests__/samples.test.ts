import { CLIEngine } from 'eslint';
import path from 'path';

const cli = new CLIEngine({
  cwd: path.resolve(__dirname, '../'),
  extensions: ['ts', 'tsx'],
});

it('snapshot', () => {
  const report = cli.executeOnFiles(['./samples/']);
  report.results.forEach((result) => {
    delete result.source;
    delete result.filePath;
  });
  expect(report).toMatchSnapshot();
});
