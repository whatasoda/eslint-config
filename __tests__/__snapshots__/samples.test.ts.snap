// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Object {
  "errorCount": 2,
  "fixableErrorCount": 0,
  "fixableWarningCount": 0,
  "results": Array [
    Object {
      "errorCount": 2,
      "filePath": "/Users/whatasoda/Documents/eslint-config/samples/index.ts",
      "fixableErrorCount": 0,
      "fixableWarningCount": 0,
      "messages": Array [
        Object {
          "column": 7,
          "endColumn": 13,
          "endLine": 7,
          "line": 7,
          "message": "'unused' is assigned a value but never used.",
          "nodeType": "Identifier",
          "ruleId": "no-unused-vars",
          "severity": 2,
        },
        Object {
          "column": 7,
          "endColumn": 13,
          "endLine": 7,
          "line": 7,
          "message": "'unused' is assigned a value but never used.",
          "nodeType": "Identifier",
          "ruleId": "@typescript-eslint/no-unused-vars",
          "severity": 2,
        },
      ],
      "warningCount": 0,
    },
  ],
  "usedDeprecatedRules": Array [],
  "warningCount": 0,
}
`;
